tests/system/Security/SecurityTest.php

```diff
@@ -209,12 +209,12 @@ public function testSendingCookiesFalse(): void
 		$request = $this->createMock(IncomingRequest::class);
 		$request->method('isSecure')->willReturn(false);
 
-		$config = new MockAppConfig();
+		$config = config('Cookie');
```

Comment:
I'm not sure if we have proper cleanup of shared instances in the tests, but please refrain from using shared instances. Instead, use the `injectMock` facility of `Factories` with a subsequent `reset` to clean up.

The thing is using `config(...)` calls in the tests and altering the state will persist and affect the other tests depending on another `config(..)` calls. As much as possible, we want to make sure each and every test receives a fresh, unadulterated instance of a config it needs.